### PR TITLE
Clarifying adding a certificate

### PR DESCRIPTION
**Pull request by bergerhoffer**
Per @sttts 

Preview: https://deploy-preview-34164--osdocs.netlify.app/openshift-enterprise/latest/security/certificates/api-server.html

### CODE REVIEW

File: security/certificates/api-server.adoc [hunk 1]
```diff
@@ -5,9 +5,11 @@ include::modules/common-attributes.adoc[]
...
 
-The default API server certificate is issued by an internal {product-title}
-cluster CA. Clients outside of the cluster will not be able to verify the
-API server's certificate by default. This certificate can be replaced
-by one that is issued by a CA that clients trust.
+The default API server certificate is issued by an internal {product-title} cluster CA. Clients outside of the cluster cannot verify the API server's certificate by default. You can add a certificate, which is issued by a CA that clients trust, for the domains specified in your certificate to use.
+
+[NOTE]
+====
+The `oc login` commands displayed from the web console during authentication always use the self-signed CA that matches the external self-signed load balancer certificate that is generated and rotated by the Kubernetes API Server Operator. It is currently not possible to override this with a custom domain.
```
Comment (mburke5678) [inline security/certificates/api-server.adoc:12]: I am not sure what this means: _The `oc login` commands displayed from the web console during authentication_. Wil this be obvious to a typical user?

**Comment by mburke5678** [comment]
@bergerhoffer One question, likely from my ignorance. Otherwise LGTM

File: security/certificates/api-server.adoc (hunk 1 above)
Comment (bergerhoffer) [reply security/certificates/api-server.adoc]: Good question, I've reworded to be clearer!

**Comment by bergerhoffer** [comment]
@sttts this was opened awhile ago per a slack conversation. Can you please take a look?

Preview: https://deploy-preview-34164--osdocs.netlify.app/openshift-enterprise/latest/security/certificates/api-server.html

File: security/certificates/api-server.adoc [hunk 2]
```diff
@@ -5,9 +5,11 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-The default API server certificate is issued by an internal {product-title}
-cluster CA. Clients outside of the cluster will not be able to verify the
-API server's certificate by default. This certificate can be replaced
-by one that is issued by a CA that clients trust.
+The default API server certificate is issued by an internal {product-title} cluster CA. Clients outside of the cluster cannot verify the API server's certificate by default. You can add a certificate, which is issued by a CA that clients trust, for the domains specified in your certificate to use.
```
Comment (sttts) [inline security/certificates/api-server.adoc:8]: would add that you cannot replace the existing cert for that domain, only add your own with another domain.

Also maybe link to the section where user certs are discussed?

**Comment by bergerhoffer** [comment]
@xingxingxia or @wangke19 Can one of you please review this update? Thanks!

Preview: https://deploy-preview-34164--osdocs.netlify.app/openshift-enterprise/latest/security/certificates/api-server.html

**Comment by wangke19** [comment]
@bergerhoffer One question is what file extension for the certificate chain? From the following, only talking it is PEM format file. 
`
Prerequisites:
...
The certificate file can contain one or more certificates in a chain. The certificate for the API server FQDN must be the first certificate in the file. It can then be followed with any intermediate certificates, and the file should end with the root CA certificate.`

But what puzzles me is that the file format here is ’.crt'. 
`
Procedure
1. Create a secret that contains the certificate chain and private key in the openshift-config namespace.
$ oc create secret tls <secret> \
     --cert=</path/to/cert.crt> \
     --key=</path/to/cert.key> \
     -n openshift-config
`
Can we clarify  in above Prerequisites : Certificate files must be Base64 PEM-encoded and typically have a .crt or .pem extension.

**Pull request closed without merging**